Gemfile.lock

```diff
@@ -1,7 +1,15 @@
+GIT
+  remote: https://github.com/aws-amplify/amplify-ci-support
+  revision: 25edc8a763327adda016ffa84ea607ca982f47e0
+  branch: master
+  glob: src/fastlane/release_actions/*.gemspec
+  specs:
+    fastlane-plugin-release_actions (0.1.0)
```

Comment:
Probably should bump this to 1.0.0 once we put it in service. I neglected to do this.

Reply:
Done.

Reply:
Gem has been updated